.github/project.yml

```
@@ -1,4 +1,4 @@
 name: Quarkus QE Test Framework
 release:
-  current-version: 1.4.2.Beta8
-  next-version: 1.4.3.Beta8
+  current-version: 1.4.4
+  next-version: 1.4.5.Beta1
```

```suggestion
  next-version: 1.4.5```

Release plugin adds `-SNAPSHOT`

---

I don't understand this tbh, what is the goal?

---

"next" imho should be 1.4.5-SNAPSHOT in pom.xml and not `1.4.5.Beta1-SNAPSHOT`
This is how to achieve it

I don't like our current `    <version>1.4.3.Beta11-SNAPSHOT</version>` in https://github.com/quarkus-qe/quarkus-test-framework/blob/main/pom.xml#L6C1-L6C45

---

That version is used just for from-branch builds, it never makes it into any release.

The downside of your approach is that the next person is going to release straight into final, that's my motivation for keeping the Beta in.

You sure you want me to remove it? I don't have a strong opinion either way, but I really don't think we should be telling people to release straight into stable release :D

---

We don't need Beta releases on micro releases

---

I've updated it :]